hooks/reviews/use-add-review.ts

```
@@ -0,0 +1,69 @@
+        queryKey: queryKeys.reviews.byUser('me'),
+      });
+      
+      toast.success('Your review has been submitted for approval');
+    },
+    onError: (error: any) => {
+      console.error('Failed to add review:', error);
+      
+      // Try to extract more detailed error information
+      let errorMessage = 'Failed to submit review. Please try again.';
+      
+      if (error.message && error.message.includes('Invalid comment')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)